### DISCUSSION TITLE
Tailwind not working on VUE APP with VITE!

### DISCUSSION THREAD

**Discussion by phantodev** [tailwindlabs/tailwindcss#4678] [Ideas]
Hi guys... can you check this link please?

https://hts-control.web.app/

In my LOCALHOST works prefect all my stylesheets... but when i run a BUILD unfortunely TAILWIND not working...

Any ideas?

**Comment by VicGUTT** [comment]
Hey !

Maybe this small Twitter thread my help you : https://twitter.com/jdlien/status/1405464914341236742
